docs/examples/modalalert/doProvideAction.js

```diff
@@ -0,0 +1,33 @@
+// @flow strict
+import { type Node } from 'react';
+import { ModalAlert, CompositeZIndex, FixedZIndex, Layer, Text } from 'gestalt';
+
+const HEADER_ZINDEX = new FixedZIndex(10);
+const zIndex = new CompositeZIndex([HEADER_ZINDEX]);
+
+export default function DoClearCommunicate(): Node {
+  return (
+    <Layer zIndex={zIndex}>
+      <ModalAlert
+        type="error"
+        accessibilityModalLabel="Unable to charge credit card"
+        heading="Your credit card has expired"
+        primaryAction={{
+          accessibilityLabel: 'Update credit card info',
+          label: 'Update credit card',
+          href: 'https://www.pinterest.com',
```

Comment:
Are we adding the visit icon to all primary actions with URLs? What;s the intention of the icon?

Reply:
The icon showcases that folks are going to be navigated away from the modal to a new page that isn't related to the page they're currently on